### PR TITLE
Video feed

### PR DESCRIPTION
**Pull request by stevenjack**
### Problem

We need to be able to update a video when admin and select if it's in the video feed or not.

### Solution

* Add additional icon next to short hash for showing that video is part of feed.
* Change endpoint and naming from caption to `details`.
* Show/Hide caption icons when setting changes.

### Notes

image

### CODE REVIEW

**Comment by stevenjack** [comment]
[[SA] update caption logic for caption + feed.](https://trello.com/c/kuVCcSvy/618-sa-update-caption-logic-for-caption-feed)

**Comment by revett** [comment]
@stevenjack is the new icon only shown to Admins?

**Comment by stevenjack** [comment]
@revett yea, let me put a screenshot on the PR

File: source/js/brands/component/video_details_save_button.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+      $('#videoDialogErrors').html('Problem saving video details, please try again').show();
+    }).always(function() {
+      body.css("cursor", "default");
+      buttons.find('input').attr("disabled", false);
+      link.val("Save");
+    });
+     
+    return false;
+  };
+
```
Comment (revett) [inline source/js/brands/component/video_details_save_button.js]: Remove single line.

File: source/js/brands/component/video_details_toogle_button.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    if (feed) {
+      feedIndicator.show();
+      $('#videoDialog .icon-feed-indicator').show();
+    } else {
+      feedIndicator.hide();
+      $('#videoDialog .icon-feed-indicator').hide();
+    }
+    obj.onDialogClose(e);
+  }
+
```
Comment (revett) [inline source/js/brands/component/video_details_toogle_button.js]: Remove single line 🏄

**Pull request merged**